sipa/static/js/register_identify.js

```diff
@@ -1,12 +1,21 @@
-function updateState() {
-    var swdd_tenant = !$("#no_swdd_tenant").prop("checked");
-    $("#tenant_number,[for=tenant_number]").toggle(swdd_tenant);
-    $("#tenant_number").prop("required", swdd_tenant);
-
-    var agdsn_history = $("#agdsn_history").prop("checked");
-    $("#previous_dorm,[for=previous_dorm]").toggle(agdsn_history);
-    $("#previous_dorm").prop("required", agdsn_history);
+function updateForm(target, invert, updateId, event) {
+    // Invert checked if invert is true (XOR)
+    const value = target.checked !== invert;
```

Comment:
Thanks!  On that note: should I use camelCase for all the variables in JavaScript?

Reply:
sure, but to be honest it doesn't really matter. I'm not quite happy with the JS (because jquery certainly allowed for easier visibility toggling), but I'm also not quite sure how to do this better. I guess the important thing is just to kick the dependency out atm.